swagger.yaml

```diff
@@ -223,28 +266,54 @@ definitions:
       gap:
         description: "Mandatory gaps in trail (must be cleared)"
         type: "boolean"
-      
+
   Trail:
     type: "object"
     required:
     - "name"
     - "location"
     properties:
-      id:
-        type: "integer"
-        format: "int64"
-      name:
-        type: "string"
-        example: "Dan's Trail"
-      difficulty:
-        type: "string"
-        enum:
-        - "Green"
-        - "Blue"
-        - "Black"
-        - "Double Black"
-      features:
-        $ref: "#/definitions/Features"
-      location:
-        description: "Zip code of trail head"
-        type: "integer"
\ No newline at end of file
+      data:
+        properties:
+          id:
+            type: "integer"
+            format: "int64"
+          type:
+            type: "string"
+            default: "Trail"
+          attributes:
+            $ref: "#/definitions/Features"
+
+  Trails:
+    type: "object"
+    properties:
+      data:
+        type: "array"
+        items:
+          properties:
+            id:
+              type: "integer"
+              format: "int64"
+            type:
+              type: "string"
+              default: "Trail"
+            attributes:
+              $ref: "#/definitions/Features"
+
+  Error:
+    properties:
+      status:
+        type: integer
+        description: HTTP status code.
+      developerMessage:
+        type: string
+        description: An error string aimed at developers.
+      userMesage:
+        type: string
+        description: An error string aimed at end users.
+      code:
+        type: integer
+        description: Error code.
+      details:
+        type: string
+        description: A link to further information about the error.
```

Comment:
Not super important, but since you put quotes around all of your previous values, you might want to do the same in the Error definition just for consistency.